### ISSUE TITLE
Early Career prevents prediction/nodes

### ISSUE THREAD

**Issue by ElWanderer** [ElWanderer/kOS_scripts#142]
kOS will crash if you try to call `POSITIONAT` (or other, related functions) before we can predict trajectories. I think that is down to whether the tracking station has been updated to level 2 or not and that we can check if that is available by seeing if `Career:PATCHLIMIT` is greater than zero.

Similarly, we are not allowed to create manoeuvre nodes until both the tracking station and mission control have been updated. That can be checked through `Career:CANMAKENODES`

Having made various Module Manager patches to allow easier construction of non-crewed missions, it is a pain that there is still a lot of manual flying. It would be nice to have some basic scripts for handling that. Getting into orbit needs a change to circularisation, for example, to do without both predictions and nodes.

**Comment by ElWanderer** [comment]
Where can we remove the requirement for nodes/`POSITIONAT`/`VELOCITYAT`?

Launch currently requires both for circularisation. We can calculate the basic velocity change without `VELOCITYAT`/`POSITIONAT`, but if we can't create a node, we would need to have a fake node that we can someone burn as if it were a real one, which sounds a bit complicated and a bit of a cheaty way around the restriction. We could switch to single-burn-to-orbit.

For the Kerbin orbit tour, deorbit is (naturally) done through a node though this is created without the prediction functions. We could possibly just work out when the node would normally be and burn retrograde at that point until the periapsis drops below 30km.

Making a script that has those two changes built-in would allow launching someone or something into orbit, then recovering them. In stock, that would hopefully be enough to get the things required for unlocking nodes/predictions.

Going further than that would require a lot more effort!

**Comment by ElWanderer** [comment]
A thought - the bits of launch/deorbit that use these abilities could be put into two separate files. Whatever new code is added to be able to do these without predictions/nodes could be put into two more files. Then we'd load the appropriate files on boot-up.

**Comment by ElWanderer** [comment]
Known values:
Current altitude
Current velocity
Apoapsis
Periapsis
Body radius
Body Mu

v^2 = Mu * ((2/r) - (1/a))

velocity at apoapsis
v0 = sqrt(Mu * ((2/r) - (1/a)))
where r = apoapsis + body radius
      a = (r + periapsis + body radius) / 2

velocity for circular orbit at apoapsis
v1 = sqrt(Mu * (1/r))

v1-v0 = size of circularisation burn

If we didn't know apoapsis/periapsis, we could probably calculate it based on radius, velocity and flight angle, though I think this data is always available.
